RecoHI/HiJetAlgos/python/HiRecoJets_cff.py

```diff
@@ -65,24 +65,15 @@
     rParam       = cms.double(0.5)
     )
 akPu5CaloJets.radiusPU = 0.5
-akPu5CaloJets.puPtMin  = cms.double(10)
-
-akPu7CaloJets = cms.EDProducer(
-    "FastjetJetProducer",
-    HiCaloJetParameters,
-    AnomalousCellParameters,
-    MultipleAlgoIteratorBlock,
-    jetAlgorithm = cms.string("AntiKt"),
-    rParam       = cms.double(0.7)
-    )
-akPu7CaloJets.radiusPU = 0.7
+akPu5CaloJets.puPtMin  = 10
 
 akPu1CaloJets = akPu5CaloJets.clone(rParam = 0.1, puPtMin = 4)
 akPu2CaloJets = akPu5CaloJets.clone(rParam = 0.2, puPtMin = 4)
 akPu3CaloJets = akPu5CaloJets.clone(rParam = 0.3, puPtMin = 6)
 akPu4CaloJets = akPu5CaloJets.clone(rParam = 0.4, puPtMin = 8)
 akPu6CaloJets = akPu5CaloJets.clone(rParam = 0.6, puPtMin = 12)
 akPu7CaloJets = akPu5CaloJets.clone(rParam = 0.7, puPtMin = 14)
+akPu7CaloJets.radiusPU = 0.7
```

Comment:
This changes the config for the previous `akPu7CaloJet` cloned from `akPu5CaloJets`.
Before signing this off for reco I would like to get the opinion of the reco HI contact (@mandrenguyen ) abouth which is of the previous two instances of `akPu7CaloJets` has to be taken. (If I have to bet, I'd rather remove this line and stick on the original implicit value of `radiusPU = 0.5`, defined at line 67 above)

Reply:
Thanks @perrota . Okay then I will update it after getting feedback from @mandrenguyen .

Reply:
@perrotta  is correct, the radiusPU should be independent of the R value used in the jet clustering.  The puPtMin is a parameter that has to be optimized for each radius.  Currently, we only run akPu4CaloJets, so it's important to keep this parameter with a value of 8.  For the other radii, we should keep the default values you've indicated, although they would probably need to be reoptimized if we were to use these radii again.

Reply:
@mandrenguyen Thank you for your feedback. I removed line 76.